eng/imports/HostAgnostic.props

```diff
@@ -36,12 +36,16 @@
 
     <!-- CPS -->
     <PackageReference Include="Microsoft.VisualStudio.ProjectSystem" />
+    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.Implementation" />
+    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.SDK" />
```

Comment:
If you use the SDK metapackage, you should be able to remove the package reference to Microsoft.VisualStudio.ProjectSystem (as the SDK metapackage links to both)